File: src/main/java/tech/zolhungaj/amqapi/servercommands/objects/AvatarDriveContributionCombo.java
package tech.zolhungaj.amqapi.servercommands.objects;

import com.squareup.moshi.Json;

public record AvatarDriveContributionCombo(

	@Json(name = "avatarName")
	String avatarName,

	@Json(name = "amount")
	double amount,

	@Json(name = "username")
	String username
) {
}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/objects/AvatarDriveContributionCombo.kt
package tech.zolhungaj.amqapi.servercommands.objects

import com.squareup.moshi.Json

@JvmRecord
data class AvatarDriveContributionCombo(
    @Json(name = "avatarName") val avatarName: String,

    @Json(name = "amount") val amount: Double,

    @Json(name = "username") val username: String
) 